Solution.yaml

```diff
@@ -27,26 +27,46 @@ parameters:
       fr: 'Symbole de la monnaie'
       en: 'Currency symbol'
     varType: 'enum'
+    options:
+      tooltipText:
+        fr: "Choisissez un\nsymbole :\n\t$\n\t€\n\t฿\n\t¥"
+        en: "Choose a\nsymbol:\n\t$\n\t€\n\t฿\n\t¥"
   - id: 'currency_name'
     labels:
       fr: 'Nom de la monnaie'
       en: 'Currency name'
     varType: 'string'
+    options:
+      tooltipText:
+        fr: 'Choisissez une monnaie'
+        en: 'Choose a currency name'
```

Comment:
Some tooltips do not really give much new information, we should probably remove them in this case (we don't need to use tooltips for all scenario parameters to illustrate the feature: we should probably limit tooltips to when extra information is needed, such as enum values and min/max values)

Reply:
The ticket require we set a tooltip for at least each basic type